### PR TITLE
Moving some URLs to the new classic

### PR DESCRIPTION
**Pull request by alfiedotwtf**
Moving some to the classic URL

### CODE REVIEW

**Comment by alfiedotwtf** [comment]
There are a lot of URLs still needing to be changed e.g github and core docs. Let me know, and i'll do the heavy lifting

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Bitcoin Core integration/staging tree
 
 [![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
 
-https://www.bitcoin.org
+https://www.bitcoinclassic.com
```
Comment (orangeman) [inline README.md]: ACK

**Comment by jtoomim** [comment]
@ahmedbodi 

@alfiedotwtf It might be best to submit these as a PR into Ahmed's branch in #2 so we can keep everything in one place and not deal with merge conflicts.

**Comment by ahmedbodi** [comment]
agreed with @jtoomim. NACK on this one. make them as a PR into my branch then i'll ACK it

**Comment by alfiedotwtf** [comment]
No problem. Will add them tonight

**Comment by alfiedotwtf** [comment]
Botched the PR. Moving new PR to #18

**Comment by orangeman** [comment]
also related to https://github.com/bitcoinclassic/bitcoinclassic/pull/24

**Pull request closed without merging**